modules/gmysqlbackend/gmysqlbackend.cc

```diff
@@ -107,6 +107,8 @@ class gMySQLFactory : public BackendFactory
     declare(suffix, "supermaster-query", "", "select account from supermasters where ip=? and nameserver=?");
     declare(suffix, "supermaster-name-to-ips", "", "select ip,account from supermasters where nameserver=? and account=?");
     declare(suffix, "supermaster-add", "", "insert into supermasters (ip, nameserver, account) values (?,?,?)");
+    declare(suffix, "supermaster-remove", "", "delete from supermasters where ip = ? and nameserver = ?");
+    declare(suffix, "supermasters-list", "", "select ip,nameserver,account from supermasters");
```

Comment:
other "list" queries use a name format of `list-somethings-...` instead of `somethings-...-list`